backend/README.md

```diff
@@ -32,13 +32,48 @@ pip install -r requirements.txt
 Rosalution uses Environment variables for configuring the application at the time
 of startup.
 
+- **ROSALUTION_ENV** Sets whether the application's environment is in production. This will run the backend with the
+[-O flag](https://docs.python.org/3/using/cmdline.html#cmdoption-O) which will turn off `__debug__` statements
+within the backend codebase.
```

Comment:
```suggestion
- **ROSALUTION_ENV** Sets whether the application's environment is in production. This will run the backend with the
[-O flag](https://docs.python.org/3/using/cmdline.html#cmdoption-O) which will turn off `__debug__` statements
within the backend codebase when using the 'entrypoint-init.sh` to start the applicaiton.
```